.travis.yml

```diff
@@ -32,6 +32,7 @@ script:
     - python setup.py develop
     - py.test
     - nosetests
+    - mypy toolz/recipes.py
```

Comment:
only checking the file I modified added type hints to. But it depends on other files: this explains why I had to ignore some errors / make mypy happy elsewhere.